src/routes/blogs.rs

```diff
@@ -100,25 +101,29 @@ pub fn new(rockets: PlumeRocket, _user: User) -> Ructe {
 #[get("/validate/<validation_id>")]
 pub fn domain_validation(
     validation_id: String,
-    valid_domains: State<HashMap<&str, Instant>>,
-) -> Result<String, String> {
-    let value = valid_domains.inner().get(validation_id.as_str());
+    valid_domains: State<Mutex<HashMap<&str, Instant>>>,
+) -> Status {
+    let mutex = valid_domains.inner().lock();
+    let mut validation_map = mutex.unwrap();
+    let validation_getter = validation_map.clone();
+
+    let value = validation_getter.get(validation_id.as_str());
     if value.is_none() {
-        return Err(String::from("404"));
+        return Status::new(404, "validation id not found");
```

Comment:
I believe there are consts that already contains standards http status in the module